src/components/FormLabel/FormLabel.tsx

```diff
@@ -16,34 +16,58 @@ interface Props {
    * Custom class to pass to label element.
    */
   className?: string;
+  /**
+   * Display label inline with surrounding elements
+   */
+  displayInline?: boolean,
   /**
    * Mark the label has invalid
    */
   hasError?: boolean;
-   /**
+  /**
    * Mark the label has disabled
    */
-  isDisabled?: boolean;
+  isDisabled?: boolean,
```

Comment:
@JCarmine in an inteface these should end in semicolons, not commas, unless I'm missing something.

Reply:
Actually, since we are no longer writing JS, we can do what ever we want! Comma, semi, or nothing, anything goes! What's with programming languages mandating a specific syntax to function properly anyways?

But in all seriousness, this rule wasn't being enforced out of the box for whatever reason, so we had inconsistencies in other files as well. Added a rule for it to `.eslintrc`. Feel free to tweak.

Reply:
Perfect, thanks! @JCarmine